### PR TITLE
Add PHP 8 support

### PR DESCRIPTION
**Pull request by MyZik**
Hi @noplanman , thank you for your work!

This PR provides a PHP 8 support and I've updated snake_case variables to camelCase, I think it looks more nicer :)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -24,11 +24,12 @@
         }
     ],
     "require": {
-        "php": "^7.0"
```
Comment (noplanman) [inline composer.json:27]: We still need to support PHP 7 here. Following the core library, let's go for `"php": "^7.3|^8.0"` here 👍

File: .travis.yml [hunk 2]
```diff
@@ -7,11 +7,11 @@ cache:
     - "$HOME/.composer/cache"
 
 php:
-  - 7.0
```
Comment (noplanman) [inline .travis.yml:10]: Add PHP 7.3 here please.

File: src/InlineKeyboardPagination.php [hunk 3]
```diff
@@ -2,6 +2,8 @@
 
 namespace TelegramBot\InlineKeyboardPagination;
 
+use JetBrains\PhpStorm\ArrayShape;
```
Comment (noplanman) [inline src/InlineKeyboardPagination.php:5]: Not entirely sure how I feel about these, but I'm open to giving it a try and keeping this in 😊

**Review by noplanman — CHANGES_REQUESTED** [review]

**Comment by noplanman** [comment]
@MyZik Thanks for the PR!

It's been a while since I've taken care of this mini component, so thanks for giving it some of love 🙏

File: src/InlineKeyboardPagination.php (hunk 3 above)
Comment (MyZik) [reply src/InlineKeyboardPagination.php]: Done :)

File: .travis.yml [hunk 4]
```diff
@@ -30,7 +32,7 @@ install:
 
 script:
   - composer check-code
-  - if [ "$TRAVIS_PHP_VERSION" == "7.1" ]; then composer test-cov; else composer test; fi
```
Comment (noplanman) [inline .travis.yml:33]: You can leave it fixed at `== "8.0"` here and below 👍 

(not sure if it even makes sense to send multiple coverage reports)

File: src/InlineKeyboardPagination.php [hunk 5]
```diff
@@ -208,22 +215,27 @@ public function getNumberOfPages(): int
...
 
     /**
      * @inheritdoc
      * @throws InlineKeyboardPaginationException
      */
-    public function getPagination(int $selected_page = null): array
-    {
-        if ($selected_page !== null) {
-            $this->setSelectedPage($selected_page);
+    public function getPagination(
```
Comment (noplanman) [inline src/InlineKeyboardPagination.php:234]: Leave a 1 single line

File: src/InlineKeyboardPagination.php [hunk 6]
```diff
@@ -333,10 +347,11 @@ protected function generateRange(): array
      *
      * @return array
      */
-    protected function generateButton(int $page): array
-    {
+    protected function generateButton(
```
Comment (noplanman) [inline src/InlineKeyboardPagination.php:350]: Leave a 1 single line

**Review by noplanman — CHANGES_REQUESTED** [review]
@MyZik Thanks for the changes. As a last thing, you can also update the readme and make the variables there `$camelCase` 💪

**Comment by noplanman** [comment]
Awesome work, thanks again sooo much @MyZik 🙏

**Review by noplanman — APPROVED** [review]

**Pull request merged**